pupa/scrape/event.py

```diff
@@ -42,8 +43,19 @@ def add_entity(self, name, entity_type, *, id, note):
             "entity_type": entity_type,
             "note": note
         }
-        if id:
-            ret['id'] = id
+        # overwrite the id that exists
+        if entity_type:
```

Comment:
`add_entity` is basically a private method, and it is always given an `entity_type`, so I think this `if` is unnecessary.